client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -84,6 +84,14 @@ case class ParsedDeltaSharingTablePath(
     schema: String,
     table: String)
 
+// A case class containing the parsed response of delta sharing rpcs.
```

Comment:
Can we document this more; especially the assumptions relationship between includedEndStreamAction, lines, and capabilities?